### DISCUSSION TITLE
Error in CubeIDE

### DISCUSSION THREAD

**Discussion by agganitk** [board707/DMD_STM32#49] [Q&A]
When I added the code STM32 CubeIDE. I see lot of  errors on buildup. Please suggest.
 I never used Arduino code in STM32CUBE ide, so not sure what wrong I am doing. 
![image](https://github.com/board707/DMD_STM32/assets/71229522/5aaf9080-41c5-4095-a0fb-f4119e11a71a)

**Comment by board707** [comment]
The library is compatible with Roger Clark's Arduino STM32 core **ONLY.**
Please see the [Readme](https://github.com/board707/DMD_STM32/tree/dev-V2#readme) on the main page of the repository.
